cmd/openshift-install/log-config.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"os"
+	"path"
+
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+type LogConfig struct {
+	// Fields contains user-provided fields to be added to all log entries.
+	// +optional
+	Fields logrus.Fields `json:"fields,omitempty"`
+
+	// Level sets the level of logging to standard out.
+	// Valid values: Info, Debug, Warn, Error
+	// +optional
+	Level string `json:"level,omitempty"`
+}
+
+func readLogConfigFile(directory string) (logConfig LogConfig, err error) {
+	file, err := os.ReadFile(path.Join(directory, "log-config.yaml"))
+	if err != nil {
+		return
+	}
+	err = yaml.Unmarshal(file, &logConfig)
```

Comment:
- check err?
- How about defaulting `level`?
```suggestion
	// Default log level
	logConfig.level = "debug"
	err = yaml.Unmarshal(file, &logConfig)
```

Reply:
err is returned here and the parent function should handle it.
I'll fix the default level.